sdk/azidentity/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/Azure/azure-sdk-for-go/sdk/azidentity
 
 go 1.18
 
+replace github.com/Azure/azure-sdk-for-go/sdk/azcore => ../azcore
```

Comment:
What's the POR to release `azcore` so we can remove this?

Reply:
Nothing out of the ordinary, just release `azcore` first. `azidentity` is the only other module that references this field.

Reply:
Yeah I saw that, no codegen changes (hooray).